README.md

```diff
@@ -1,46 +1,91 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<!-- TABLE OF CONTENTS -->
 
-# Portfolio: Mobile first
+# 📗 Table of Contents
 
-> This project is a porfolio project based on CSS Mobile First concept.  
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
-## Built With
+# 📖 Portfolio: Mobile first <a name="about-project"></a>
 
-- HTML5 & CSS3
+> This project is based on CSS Mobile First concept.
 
+**Portfolio: Mobile first** is a Portfolio web page to show case my web developement skills.
 
-## Live Demo
+<!-- BUILT WITH -->
 
-[Live Demo Link](https://mnisarali.github.io/Portfolio-project/)
+## 🛠 Built With <a name="built-with"></a>
 
+- **HTML5** 
+- **CSS3**
+- **JavaScript**
 
-## Authors
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link](https://mnisarali.github.io/Portfolio-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
 
 👤 **M Nisar Ali**
 
 - GitHub: [@MNisarAli](https://github.com/MNisarAli)
 - Twitter: [@drnisar_ali](https://twitter.com/drnisar_ali)
 - LinkedIn: [Muhammad Nisar Ali](https://linkedin.com/in/muhammad-nisar-ali-45a865251)
 
+👤 **Mohammad Suliman Joya**
 
-## 🤝 Contributing
+- GitHub: [@SulimanJoya](https://github.com/SulimanJoya)
+- LinkedIn: [sjoya66](https://www.linkedin.com/in/sjoya66/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](https://github.com/MNisarAli/Portfolio-project/issues/).
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please give a ⭐️ to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Show your support
+<!-- ACKNOWLEDGEMENTS -->
 
-Give a ⭐️ if you like this project!
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
+> I would like to thank [Microverseinc](https://github.com/microverseinc).
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Inspiration: [@microverseinc](https://github.com/microverseinc)
+<!-- LICENSE -->
 
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
 
-This project is [MIT](./LICENSE) licensed.
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
All the main headers from [this README template](https://github.com/microverseinc/readme-template) which are relevant to your project should be present in your repo. The only optional section is FAQ. Kindly make sure your readme also has added the section **Future Features**, including 1-3 potential features to be added, and also the **Key Features** section. You can remove sections like **Tech Stack** or some parts of the **Getting Started** section.

Reply:
Thanks for your review. I've made the required changes in `README`.